src/helpers/get-vulcan-presets.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Returns all presets of a specific type.
+ *
+ * @param {'deliver'|'compute'} type - The type of preset to return
+ * @return {string[]} List of presets of the specified type
+ */
+export const getVulcanPresets = (type = null) => {
+  const DELIVER_PRESETS = ['nextjs', 'angular', 'astro', 'hexo', 'react', 'vite', 'vue']
+  const COMPUTE_PRESETS = ['nextjs']
+
+  if (type === 'deliver') {
+    return DELIVER_PRESETS
+  }
+
+  if (type === 'compute') {
+    return COMPUTE_PRESETS
+  }
+
+  const allPresets = [...DELIVER_PRESETS, ...COMPUTE_PRESETS]
+  return allPresets
```

Comment:
sugestão:
```suggestion
  const DELIVER_PRESETS = ['nextjs', 'angular', 'astro', 'hexo', 'react', 'vite', 'vue']
  const COMPUTE_PRESETS = ['nextjs']
  const ALL_PRESETS = [...DELIVER_PRESETS, ...COMPUTE_PRESETS]
  
  const presets = {
    deliver:  DELIVER_PRESETS,
    compute: COMPUTE_PRESETS
   }

    return presets[type] || ALL_PRESETS
```

A busca é tão rápida quanto a condicional e o método fica aberto para futuras adições que não demandam mais condições

Reply:
Sugestão aderida, mestre! Muito boa.